fixtures/blockers.py

```diff
@@ -49,3 +50,47 @@ def bug(blocker):
         Instance of :py:class:`utils.bz.BugWrapper` or :py:class:`NoneType` if the bug is closed.
     """
     return lambda bug_id: blocker("BZ#{}".format(bug_id)).bugzilla_bug
+
+
+def pytest_addoption(parser):
+    group = parser.getgroup('Blockers options')
+    group.addoption('--disable-blocker-listing',
+                    action='store_false',
+                    default=True,
+                    dest='list_blockers',
+                    help='Specify to disable listing of the blockers to speed up startup time.')
+
+
+@pytest.mark.trylast
+def pytest_collection_modifyitems(session, config, items):
+    if not config.getvalue("list_blockers"):
+        return
+    store.terminalreporter.write("Loading blockers ...\n", bold=True)
+    blocking = set([])
+    for item in items:
+        if "blockers" not in item._metadata:
+            continue
+        for blocker in item._metadata["blockers"]:
+            if isinstance(blocker, int):
+                # TODO: DRY
+                blocker_object = Blocker.parse("BZ#{}".format(blocker))
+            else:
+                blocker_object = Blocker.parse(blocker)
+            if blocker_object.blocks:
+                blocking.add(blocker_object)
+    store.terminalreporter.write("Known blockers:\n", bold=True)
```

Comment:
This prints even if there are no blockers, should probably be wrapped in an `if`, saying this if there are blockers or otherwise saying there are none.